### DISCUSSION TITLE
Add a manifest.json to make saving the the Home Screen on iOS more app like

### DISCUSSION THREAD

**Discussion by robacarp** [AllYourBot/hostedgpt#224] [General]
I’ve done some of this before on my own home grown app-ish things and it make the experience so much more appealing. Adding it to the app switcher instead of safari tabs, etc.

**Comment by keithschacht** [comment]
Nice, I’ve not built this before but I knew it was some special configuration. Issue #205 is my placeholder for this task. It definitely needs to be fleshed out.

But it’s important that things also work smoothly for non PWA mode where there is an address bar / header that we have to contend with. Separate work streams but maybe a little related.
